setup.py

```diff
@@ -124,15 +124,11 @@ def run(self):
     "numpy>=1.12.0",
 ]
 
-with open('VERSION') as versionf:
-    version = versionf.readline().strip()
-
 setup(
     name="libsonata",
     description='SONATA files reader',
     author="BlueBrain Project, EPFL",
     author_email="bbp-ou-nse@groupes.epfl.ch",
-    version=version,
```

Comment:
is `use_scm_version=True,` the default now?

Reply:
Right, forgot about that. Would explain why travis sees 0.0.0 ☺

Reply:
And travis picks up a better version number. I'd like to get this in and then tag `0.1.0`, if that's ok.

Reply:
yup, sounds good